### ISSUE TITLE
BigInt literals are not available when targeting lower than ES2020

### ISSUE THREAD

**Issue by ivpusic** [pimlicolabs/permissionless.js#99]
Hi,

It would be cool if you can convert bigint literals to BigInt(<number>) form, so targets lower than ES2020 are supported.

Example:
```typescript
const multiplier = userOperation.paymasterAndData.length > 2 ? 3n : 1n
```

can be converted to:
```typescript
const multiplier = userOperation.paymasterAndData.length > 2 ? BigInt(3) : BigInt(1)
```

**Comment by kristofgazso** [comment]
@plusminushalf

**Comment by plusminushalf** [comment]
Fix - https://github.com/pimlicolabs/permissionless.js/pull/168

**Comment by plusminushalf** [comment]
FIxed, Thanks for this!

**Comment by plusminushalf** [comment]
Released fix in `permissionless@0.1.17`
